recipes-kernel/linux/linux-sunxi_3.4.bb

```diff
@@ -3,10 +3,11 @@ require linux.inc
 DESCRIPTION = "Linux kernel for Allwinner a10/a20 processors"
 
 COMPATIBLE_MACHINE = "(sun4i|sun5i|sun7i)"
+DEPENDS = "sunxi-board-fex"
 
-PV = "3.4.103"
+PV = "3.4.104"
 PR = "r1"
```

Comment:
May be put the version bump in a separate commit/pull request? I'm quite new to the meta-sunxi community so I'm not sure what the gudelines are around here.